shared/scripts/virtio_console_guest.py

```diff
@@ -273,7 +291,10 @@ def _get_port_status(self, in_files=None):
                         port[m.group(1)] = m.group(2)
 
                     if port['is_console'] == "yes":
-                        port["path"] = "/dev/hvc%s" % (port["console_vtermno"])
+                        if fix_kernel_version:
```

Comment:
Let's make code more readable,  I guess you want to change base port number by kernel.  let's give your some example code in below:

```
import platform
from distutils.version import LooseVersion
current_version = LooseVersion(platform.uname()[2])
fixed_version = = LooseVersion("4.18.0-97")
port_num = int(port["console_vtermno"])
path  = "/dev/hvc%d" % ( port_num if  current_version >= fixed_version  else port_num -1 )
```

Reply:
Thanks and update, please help to review this.